### PR TITLE
update tiflash upgrade guide

### PR DESCRIPTION
**Pull request by lidezhu**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash-upgrade-guide.md [hunk 1]
```diff
@@ -82,8 +82,14 @@ TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此，从 v5.x 或 v6
...
-## 从 v6.x 或 v7.x 升级至 v7.3，并且设置了 `storage.format_version = 5`
+## 从 v6.x 或 v7.x 升级至 v7.3
 
 从 v7.3 开始，TiFlash 支持新的 DTFile 版本 V3 (实验特性），可以将多个小文件合并成一个大文件，减少文件数量。DTFile 在 v7.3 的默认版本是 V2，如需使用 V3，可通过 [TiFlash 配置参数](/tiflash/tiflash-configuration.md) `storage.format_version = 5` 来设置。设置后，TiFlash 仍可以读 V2 版本的 DTFile，并且在后续的数据整理 (Compaction) 中会将这些 V2 版本的 DMFile 逐步重新写为 V3 版本的 DTFile。
 
-在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
\ No newline at end of file
+在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
+
+从 v7.3 开始，PageStorage V3 底层部分存储文件名发生改动，导致升级到 v7.3 后，不支持原地降级到之前的版本。
```
Comment (JaySon-Huang) [inline tiflash-upgrade-guide.md:91]: this affects v7.4 since we revert the changes in v7.3

File: tiflash-upgrade-guide.md [hunk 2]
```diff
@@ -86,4 +86,12 @@ TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此，从 v5.x 或 v6
 
 从 v7.3 开始，TiFlash 支持新的 DTFile 版本 V3 (实验特性），可以将多个小文件合并成一个大文件，减少文件数量。DTFile 在 v7.3 的默认版本是 V2，如需使用 V3，可通过 [TiFlash 配置参数](/tiflash/tiflash-configuration.md) `storage.format_version = 5` 来设置。设置后，TiFlash 仍可以读 V2 版本的 DTFile，并且在后续的数据整理 (Compaction) 中会将这些 V2 版本的 DMFile 逐步重新写为 V3 版本的 DTFile。
 
-在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
\ No newline at end of file
+在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
+
+## 从 v6.x 或 v7.x 升级至 v7.4
+
+从 v7.4 开始，PageStorage V3 底层部分存储文件名发生改动，导致升级到 v7.4 后，不支持原地降级到之前的版本。
```
Comment (JaySon-Huang) [inline tiflash-upgrade-guide.md:93]:
```suggestion
从 v7.4 开始，为了减少数据整理时产生的读、写放大，PageStorage V3 数据整理时逻辑进行了优化，导致底层部分存储文件名发生改动，因此升级到 v7.4 后，不支持原地降级到之前的版本。
```

**Review by JaySon-Huang — APPROVED** [review]
LGTM

**Review by flowbehappy — APPROVED** [review]

File: tiflash-upgrade-guide.md [hunk 3]
```diff
@@ -82,10 +82,16 @@ TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此，从 v5.x 或 v6
...
+## 从 v6.x 或 v7.x 升级至 v7.3，并且设置了 `storage.format_version = 5`
 
 从 v7.3 开始，TiFlash 支持新的 DTFile 版本 V3 (实验特性），可以将多个小文件合并成一个大文件，减少文件数量。DTFile 在 v7.3 的默认版本是 V2，如需使用 V3，可通过 [TiFlash 配置参数](/tiflash/tiflash-configuration.md) `storage.format_version = 5` 来设置。设置后，TiFlash 仍可以读 V2 版本的 DTFile，并且在后续的数据整理 (Compaction) 中会将这些 V2 版本的 DMFile 逐步重新写为 V3 版本的 DTFile。
 
-从 v7.4 开始，DTFile 版本 V3 正式 GA，成为 TiFlash DTFile 的默认版本。
+在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
 
-在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 或者升级到 v7.4 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
\ No newline at end of file
+## 从 v6.x 或 v7.x 升级至 v7.4
```
Comment (qiancai) [inline tiflash-upgrade-guide.md:91]:
```suggestion
## 从 v6.x 或 v7.x 升级至 v7.4 或以上版本
```

File: tiflash-upgrade-guide.md [hunk 4]
```diff
@@ -82,10 +82,16 @@ TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此，从 v5.x 或 v6
...
 从 v7.3 开始，TiFlash 支持新的 DTFile 版本 V3 (实验特性），可以将多个小文件合并成一个大文件，减少文件数量。DTFile 在 v7.3 的默认版本是 V2，如需使用 V3，可通过 [TiFlash 配置参数](/tiflash/tiflash-configuration.md) `storage.format_version = 5` 来设置。设置后，TiFlash 仍可以读 V2 版本的 DTFile，并且在后续的数据整理 (Compaction) 中会将这些 V2 版本的 DMFile 逐步重新写为 V3 版本的 DTFile。
 
-从 v7.4 开始，DTFile 版本 V3 正式 GA，成为 TiFlash DTFile 的默认版本。
+在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
 
-在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 或者升级到 v7.4 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
\ No newline at end of file
+## 从 v6.x 或 v7.x 升级至 v7.4 或以上版本
+
+从 v7.4 开始，为了减少数据整理时产生的读、写放大，PageStorage V3 数据整理时逻辑进行了优化，导致底层部分存储文件名发生改动，因此升级到 v7.4 后，不支持原地降级到之前的版本。
```
Comment (qiancai) [inline tiflash-upgrade-guide.md:93]:
```suggestion
从 v7.4 开始，为了减少数据整理时产生的读、写放大，PageStorage V3 数据整理时逻辑进行了优化，导致底层部分存储文件名发生改动，因此升级到 v7.4 或以上版本后，不支持原地降级到之前的版本。
```

File: tiflash-upgrade-guide.md [hunk 5]
```diff
@@ -82,10 +82,16 @@ TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此，从 v5.x 或 v6
...
 从 v7.3 开始，TiFlash 支持新的 DTFile 版本 V3 (实验特性），可以将多个小文件合并成一个大文件，减少文件数量。DTFile 在 v7.3 的默认版本是 V2，如需使用 V3，可通过 [TiFlash 配置参数](/tiflash/tiflash-configuration.md) `storage.format_version = 5` 来设置。设置后，TiFlash 仍可以读 V2 版本的 DTFile，并且在后续的数据整理 (Compaction) 中会将这些 V2 版本的 DMFile 逐步重新写为 V3 版本的 DTFile。
 
-从 v7.4 开始，DTFile 版本 V3 正式 GA，成为 TiFlash DTFile 的默认版本。
+在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
 
-在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 或者升级到 v7.4 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
\ No newline at end of file
+## 从 v6.x 或 v7.x 升级至 v7.4 或以上版本
+
+从 v7.4 开始，为了减少数据整理时产生的读、写放大，PageStorage V3 数据整理时逻辑进行了优化，导致底层部分存储文件名发生改动，因此升级到 v7.4 或以上版本后，不支持原地降级到之前的版本。
```
Comment (qiancai) [inline tiflash-upgrade-guide.md:93]:
```suggestion
从 v7.4 开始，为了减少数据整理时产生的读、写放大，PageStorage V3 数据整理时逻辑进行了优化，导致底层部分存储文件名发生改动。因此，升级 TiFlash 到 v7.4 或以上版本后，不支持原地降级到之前的版本。
```

File: tiflash-upgrade-guide.md [hunk 6]
```diff
@@ -82,10 +82,16 @@ TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此，从 v5.x 或 v6
...
 
-在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 或者升级到 v7.4 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
\ No newline at end of file
+## 从 v6.x 或 v7.x 升级至 v7.4 或以上版本
+
+从 v7.4 开始，为了减少数据整理时产生的读、写放大，PageStorage V3 数据整理时逻辑进行了优化，导致底层部分存储文件名发生改动。因此，升级 TiFlash 到 v7.4 或以上版本后，不支持原地降级到之前的版本。
+
+**测试环境及特殊回退需求下的对策**
+
+强制缩容 TiFlash 节点，并重新同步数据。操作步骤详见[缩容 TiFlash 节点](/scale-tidb-using-tiup.md#缩容-tiflash-节点)。
```
Comment (hfxsd) [inline tiflash-upgrade-guide.md:97]:
```suggestion
如果在测试环境下、或者其他有回退需求的场景下，可以强制缩容 TiFlash 节点，并重新同步数据。操作步骤详见[缩容 TiFlash 节点](/scale-tidb-using-tiup.md#缩容-tiflash-节点)。
```

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
